README.md

```diff
@@ -53,9 +53,31 @@ Combining `editor.suggest.preview` with the [pre-fill required fields](#code-com
 
 Completing the snippet allows you to tab complete through each attribute and block.
 
-### Syntax Validation
+### Validation
+
+While editing, the configuration is validated and invalid code is highlighted. 
```

Comment:
```suggestion
While editing, the configuration is validated and invalid code is highlighted. 
Terraform configuration files are validated when on opened and on change, and invalid code is marked with diagnostics.
```

Active versus passive tone is preferred in general. I switched higlighted to marked as we technically do not highlight words, but use the built in diagnostics framework of underlines and problems pane.